h/templates/layouts/account.html.jinja2

```diff
@@ -15,23 +15,45 @@
 {% block content %}
   <div class="content paper">
     {% include "h:templates/includes/logo-header.html.jinja2" %}
-    <ul class="nav nav-tabs">
-      {% for route, title in nav_pages %}
-        <li{% if route == page_route %} class="active"{% endif %}>
-            <a href="{{ request.route_url(route) }}">{{ title }}</a>
-        </li>
-      {% endfor %}
-    </ul>
-
-    {% if request.session.peek_flash('success') -%}
-    <div class="form-flash">
-      {% for message in request.session.pop_flash('success') %}
-        <p>{{ message }}</p>
-      {%- endfor %}
-    </div>
-    {%- endif %}
-
-    {% block page_content %}
-    {% endblock page_content %}
+    {% if feature('activity_pages') %}
+      <div class="form">
```

Comment:
So we're using `<div class="form">` in order to get the horizontal alignment right here, but the navigation is clearly not a form.

I'm going to defer to @robertknight on the right way to solve this one, but IMO we definitely shouldn't be using CSS classes which are named "form" for things that aren't forms.